package.json

```diff
@@ -137,12 +137,16 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "chokidar '(client|server|test)/**/*.(js|jsx)' -c 'pkill node -f runner.js; npm run test-client ${TEST_PATH}'",
```

Comment:
Cool, `nodemon` definitely looks more reliable.

Two tidbits:

- we always run the tests in the beginning – in the case of the whole suite it can be slightly unnecessary. Not sure if it’s avoidable, though.
- If we run `npm run test-client:watch client/state/posts`, but change a file outside of this directory, should we still run the tests?